proto/internal/temporal/server/api/persistence/v1/hsm.proto

```diff
@@ -161,3 +168,39 @@ message StateMachinePath {
     repeated StateMachineKey path = 1;
 }
 
+// StateMachineOperation represents a history record of operations performed on nodes in the state machine tree
+message StateMachineOperation {
```

Comment:
Not sure this should be defined here, there's already a representation of operations that was added for state based replication.

Reply:
Fixed